src/native/fs/fs_napi.cpp

```diff
@@ -149,6 +159,22 @@ struct FSWorker : public Napi::AsyncWorker
             std::string errmsg = strerror(errno);
             SetError(errmsg);
         }
+        bool change_uid = orig_uid != _req_uid;
+        bool change_gid = orig_gid != _req_gid;
+        if (change_uid) {
+            int r = syscall(SYS_setresuid, -1, orig_uid, -1);
```

Comment:
Why is this setuid/setgid needed here? Seems very unrelated to me

Reply:
right, now that it panics - this one can be removed